src/sentry/event_manager.py

```diff
@@ -556,6 +556,7 @@ def save(self, project, raw=False):
             'level': level,
             'last_seen': date,
             'first_seen': date,
+            'active_at': date,
```

Comment:
this also wasnt populating -- i couldnt find any reason why, but according to docstring it seems correct